dir.targets

```diff
@@ -59,6 +59,17 @@
     </Task>
   </UsingTask>
 
+  <UsingTask TaskName="UnzipFile" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v12.0.dll">
```

Comment:
FYI - If you plan to use this in the .NET Core version of msbuild or on mono neither support the CodeTaskFactory.

We actually have ZipFileFromDirectory build task (https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/ZipFileCreateFromDirectory.cs#L13) perhaps we should add an Extract.